docs/datamodel/links.rst

```diff
@@ -14,16 +14,265 @@ There are two kinds of link item declarations: *abstract links*, and
 *concrete links*.  Abstract links are defined on the module level and are
 not tied to any particular object type. Typically this is done to set
 some :ref:`annotations <ref_datamodel_annotations>`, define
-:ref:`link properties <ref_datamodel_props>`, or setup :ref:`constraints
-<ref_datamodel_constraints>`.  Concrete links are defined on specific object
-types.
+:ref:`link properties <ref_datamodel_props>`, or setup
+:ref:`constraints <ref_datamodel_constraints>`.  Concrete links
+are defined on specific object types.
 
 Links are directional and have a *source* and one or more *targets*.
```

Comment:
Links don't have multiple targets.  Rather, a single *source* may have multiple *links* of a kind.  It is easy to see the difference when you take link properties into account.

Reply:
OK, I reworded it to talk about sources having links of certain kinds.